database/gormdb_assignment.go

```diff
@@ -123,6 +123,19 @@ func (db *GormDB) GetCourseAssignmentsWithSubmissions(courseID uint64, submissio
 	return filteredAssignments, nil
 }
 
+// GetCourseAssignmentsWithSubmissionsNoBuildInfo
+// returns data required for results page (score and status)
+func (db *GormDB) GetCourseAssignmentsWithSubmissionsNoBuildInfo(courseID uint64, submissionType pb.SubmissionsForCourseRequest_Type) ([]*pb.Assignment, error) {
+	var assignments []*pb.Assignment
+
+	if err := db.conn.Preload("Submissions").Where(&pb.Assignment{CourseID: courseID}).Order("order").Find(&assignments).Error; err != nil {
+		fmt.Println(err.Error())
```

Comment:
Should not print errors, since it is returned and printed by the caller or some other caller up the call stack.